linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/oval/shared.xml

```diff
@@ -9,7 +9,11 @@
     <ind:object object_ref="object_no_empty_passwords" />
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="object_no_empty_passwords" version="1">
+{{% if product == "sle12" %}}
+    <ind:filepath operation="pattern match">^/etc/pam.d/.*$</ind:filepath>
+{{% else %}}
     <ind:filepath>/etc/pam.d/system-auth</ind:filepath>
+{{% endif %}}
```

Comment:
It may be good to have this behavior change reflected in the rule description to avoid confusion.

Reply:
Good point!